src/vehicle.cpp

```diff
@@ -6351,19 +6374,38 @@ bool vehicle_part::fault_set( const fault_id &f )
     return true;
 }
 
-int vehicle_part::wheel_area() const
+/** Get wheel area or return 0 if part is not wheel */
+float vehicle_part::wheel_area() const
```

Comment:
`wheel_area()` is used in one place only. I'd drop it to make the interface a wee bit narrower than it is now. Also, the condition below is useless. Both `wheel_diameter()` and  `wheel_width()` would return `0` for non-wheels.